src/app/sponsors/page.tsx

```diff
@@ -0,0 +1,77 @@
+import Paragraph from '@/components/Paragraph';
+import Sponsors from '@/components/Sponsors';
+import Title from '@/components/Title';
+import type { SponsorType } from '@/data/sponsors';
+import Image from 'next/image';
+
+const SPONSOR_TYPE_COLORS = {
+    gold: '#FCC018',
+    silver: '#C3C3C3',
+    bronze: '#E8903F',
+} as const satisfies Record<SponsorType, string>;
+
+export default function SponsorsPage() {
+    return (
+        <div className="flex flex-col items-center gap-10">
```

Comment:
You should be using semantic HTML tags like `main` and `section` : https://www.w3schools.com/html/html5_semantic_elements.asp.